### PR TITLE
Upgrade next to v15

### PR DESCRIPTION
**Pull request by pettinarip**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "clsx": "^2.1.1",
     "geist": "^1.2.1",
     "lucide-react": "^0.416.0",
-    "next": "^14",
+    "next": "^15.0.3",
```
Comment (wackerow) [inline package.json:39]: @pettinarip Hm, do we not need to update React as well?

https://nextjs.org/docs/app/building-your-application/upgrading/version-15

This is building just fine for me, but the docs suggest React 19 RC is required

Comment (pettinarip) [reply package.json:39]: Ah! thats a great point! weird that I didn't have any peer dep warning 🤔 .... let me bump them

File: package.json [hunk 2]
```diff
@@ -36,12 +36,12 @@
...
     "next-sitemap": "^4.2.3",
     "next-themes": "^0.3.0",
     "pretty-ms": "^9.2.0",
-    "react": "18.2.0",
-    "react-dom": "18.2.0",
+    "react": "18.3.1",
+    "react-dom": "18.3.1",
```
Comment (wackerow) [inline package.json:44]: https://nextjs.org/docs/app/building-your-application/upgrading/version-15#react-19
image

@pettinarip v19, no?

**Comment by wackerow** [comment]
React 19 available, but will wait until our dependencies fully support it to move forward here

**Comment by pettinarip** [comment]
Closing for now until our deps have better support for react 19.

**Pull request closed without merging**